constants.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Application Constants file
+ */
+
+const busApiMeta = {
+  originator: 'av-scanner-service',
+  mimeType: 'application/json',
+  events: {
+    scan: 'avscan.action.scan'
```

Comment:
@sharathkumaranbu This cannot be a constant since `submission-processor` has this as an environment variable

Reply:
I have introduced environment variable `AVSCAN_TOPIC` for the value `avscan.action.scan`